### PR TITLE
Mask for mdb class

### PR DESCRIPTION
**Pull request by ykawashima**
I have added ``apply_mask_mdb`` functions to MdbExomol/Hitemp/Hitran classes so that the user can apply their own mask after the mdb extraction.

For HITEMP and HITRAN, I applied the mask to the instance variables written in ``instances_from_dataframes``. For ExoMol, I applied the mask to gamma_natural, alpha_ref, n_Texp, and logsij0 in addition to those written in ``instances_from_dataframes``. (Is that sufficient? I have confirmed that the spectrum calculation works.)

You can check the update with the following script. Thank you.
```
import numpy as np
from exojax.utils.grids import wavenumber_grid
from exojax.spec import api
nus,wav,res=wavenumber_grid(6910,6990,100000,unit='cm-1',xsmode="premodit")

# ExoMol                                                                                                                      
mdb = api.MdbExomol("/home/kawashima/database/H2O/1H2-16O/POKAZATEL",nus)
print(len(mdb.elower), np.min(mdb.elower))

mask = mdb.elower > 100.
mdb.apply_mask_mdb(mask)
print(len(mdb.elower), np.min(mdb.elower))

# HITEMP                                                                                                                      
mdb = api.MdbHitemp("/home/kawashima/database/H2O/01_HITEMP2010",nus)
print(len(mdb.n_air), np.min(mdb.n_air))

mask = mdb.n_air > 0.01
mdb.apply_mask_mdb(mask)
print(len(mdb.n_air), np.min(mdb.n_air))

# HITRAN                                                                                                                      
mdb = api.MdbHitran("/home/kawashima/database/H2O/01_hit12.par",nus)
print(len(mdb.n_air), np.min(mdb.n_air))

mask = mdb.n_air > 0.01
mdb.apply_mask_mdb(mask)
print(len(mdb.n_air), np.min(mdb.n_air))
```

### CODE REVIEW

**Review by HajimeKawahara — APPROVED** [review]
Thanks! all of the unit tests passed and I confirmed your sample worked.
